crates/wash-cli/src/bin/wash.rs

```diff
@@ -220,7 +220,7 @@ enum CliCommand {
 #[tokio::main]
 async fn main() {
     use clap::CommandFactory;
-    env_logger::init();
+    tracing_subscriber::init();
```

Comment:
Just a heads up, the simplest substitution here is `tracing_subscriber::fmt::init();` That will give you basically the same behavior as `env_logger::init()`. We can add more advanced filtering and options later if we wish